tests/Unit/Services/Anonymize/AnonymizeParticipantTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Tests\Unit\Services\Anonymize;
+
+use App\Comment;
+use App\Participant;
+use App\Services\Anonymize\AnonymizeGeneratorInterface;
+use App\Services\Anonymize\AnonymizeParticipant;
+use App\Services\ObjectManager\ObjectManagerInterface;
+use App\User;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Collection;
+use Mockery;
+use Tests\TestCase;
+
+
+/**
+ * Technically not an unit test because of `test_it_gets_participants_to_anonymize` hitting the database
+ * But splitting this us was to much effort and code duplication for noww
```

Comment:
```suggestion
 * Technically not a unit test because of `test_it_gets_participants_to_anonymize` hitting the database
 * But splitting this up was too much effort and code duplication for now
```